openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -61,18 +62,39 @@ def create_jwt_from_token(token_dict, oauth_adapter, use_asymmetric_key=None):
     access_token = oauth_adapter.get_access_token(token_dict['access_token'])
     client = oauth_adapter.get_client_for_token(access_token)
 
-    # TODO (ARCH-204) put access_token as a JWT ID claim (jti)
```

Comment:
**Reviewer Note:** The ticket doesn't provide info around this, and this sounds like a solution (which I don't really agree with) to a problem that is not documented here, but I'm not even sure. So, I think it is better to remove and update when and if we have a real problem to solve.

Note: my guess that this is to potentially one day make the JWTs revocable, but I don't think that's a good idea. Better to have them be shorter lived and not to do DB checks, especially because a revoked JWT would only affect LMS.

Reply:
FYI: @dianakhuang. And thank you for the review!